tasks/copy_files_to_archive/copy_files_to_archive.py

```diff
@@ -105,92 +86,19 @@ def task(records: List[Dict[str, Any]], max_retries: int, retry_sleep_secs: floa
         if not a_file[FILE_SUCCESS_KEY]:
             any_error = True
             now = datetime.now(timezone.utc).isoformat()
-            post_status_for_file_to_queue(
-                a_file[INPUT_JOB_ID_KEY], a_file[INPUT_GRANULE_ID_KEY], a_file[INPUT_FILENAME_KEY], None,
-                None,
-                ORCA_STATUS_FAILED, a_file.get(FILE_ERROR_MESSAGE_KEY, None),
-                None, now, now, RequestMethod.PUT, db_queue_url,
-                max_retries, retry_sleep_secs)
-
+            shared_recovery.update_status_for_file(
+                a_file[INPUT_JOB_ID_KEY], a_file[INPUT_GRANULE_ID_KEY], a_file[INPUT_FILENAME_KEY], 
+                shared_recovery.OrcaStatus.FAILED.value, a_file.get(FILE_ERROR_MESSAGE_KEY, None),db_queue_url)
```

Comment:
In my nitpicky vein, there's a space missing in these params before db_queue_url.

Reply:
good catch :)